app/Repositories/LocationRepository.php

```diff
@@ -50,30 +50,15 @@ public function getData($location_id, $filter)
 
     public function getAllShift($location, $filter)
     {
+        $total_seat = $location->total_seat;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Variable "total_seat" is not in valid camel caps format